routes/search.js

```diff
@@ -29,9 +29,8 @@ module.exports = (elastic, config) => ({
                 const query = value.query;
                 const queryParams = createQueryParams('html', {query: query, params: params});
                 search(elastic, queryParams, (err, result) => {
-                  if (err) return reply(err);
+                  if (err) return reply(Boom.badRequest(err));
```

Comment:
Have you considered using `Hoek.assert` to return errors? 
see: https://github.com/dwyl/hapi-error#custom-error-messages-using-hoekassert